auth/store.go

```diff
@@ -114,7 +114,7 @@ type AuthStore interface {
 	IsRangePermitted(header *pb.RequestHeader, key, rangeEnd []byte) bool
 
 	// IsDeleteRangePermitted checks delete-range permission of the user
-	IsDeleteRangePermitted(username string, key, rangeEnd []byte) bool
+	IsDeleteRangePermitted(header *pb.RequestHeader, key, rangeEnd []byte) bool
```

Comment:
@mitake Actually I would like to change everything to (ctx context.Context, username string, key, rangeEnd []byte) style as func Authenticate and IsDeleteRangePermitted